platform/security/sts/security-sts-ldapclaimshandler/src/test/java/ddf/security/sts/claimsHandler/RoleClaimsHandlerTest.java

```diff
@@ -125,6 +129,75 @@ public void testRetrieveClaimsValuesNotNullPrincipal()
     assertThat(claim.getValues().get(0), equalTo(groupName));
   }
 
+  @Test
+  public void testRetrieveClaimsValuesNestedUserOU()
+      throws LdapException, SearchResultReferenceIOException {
+    BindResult bindResult = mock(BindResult.class);
+    ClaimsParameters claimsParameters;
+    Connection connection = mock(Connection.class);
+    ConnectionEntryReader membershipReader = mock(ConnectionEntryReader.class);
+    ConnectionEntryReader groupNameReader = mock(ConnectionEntryReader.class);
+    LDAPConnectionFactory connectionFactory = PowerMockito.mock(LDAPConnectionFactory.class);
+    LinkedAttribute membershipAttribute = new LinkedAttribute("cn");
+    LinkedAttribute groupNameAttribute = new LinkedAttribute("cn");
+    ProcessedClaimCollection processedClaims;
+    RoleClaimsHandler claimsHandler;
+    SearchResultEntry membershipSearchResult = mock(SearchResultEntry.class);
+    DN resultDN = DN.valueOf("uid=tstark,OU=nested,");
```

Comment:
Same as above.  Normally, if the base user DN is something other than blank, it would look something like `uid=tstark,OU=nested,OU=users,DC=example,DC=com` where `OU=users,DC=example,DC=com` is the base user DN.